src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3262,7 +3309,7 @@
 examples:
   - name: Update an outbound-rule. (autogenerated)
     text: |
-        az network lb outbound-rule update --enable-tcp-reset true --lb-name MyLb --name MyOutboundRule --outbound-ports 10000 --resource-group MyResourceGroup
+        az network lb outbound-rule update --lb-name MyLb --name MyOutboundRule --outbound-ports 10000 --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
one concern for --subscription.
the second issue is that if there is parameter change, we should add example not replace, unless the previous one is wrong.

Reply:
```suggestion
        az network lb outbound-rule update --lb-name MyLb --name MyOutboundRule --outbound-ports 10000 --resource-group MyResourceGroup
```